presto-verifier/src/test/java/com/facebook/presto/verifier/rewrite/TestQueryRewriter.java

```diff
@@ -76,13 +77,15 @@ public void setup()
     {
         queryRunner = setupPresto();
         queryRunner.execute("CREATE TABLE test_table (a bigint, b varchar)");
+        PrestoActionConfig prestoActionConfig = new PrestoActionConfig()
+                .setHosts(queryRunner.getServer().getAddress().getHost())
+                .setJdbcPort(queryRunner.getServer().getAddress().getPort());
         prestoAction = new JdbcPrestoAction(
                 PrestoExceptionClassifier.defaultBuilder().build(),
                 CONFIGURATION,
                 VerificationContext.create(SUITE, NAME),
-                new PrestoActionConfig()
-                        .setHosts(queryRunner.getServer().getAddress().getHost())
-                        .setJdbcPort(queryRunner.getServer().getAddress().getPort()),
+                new JdbcUrlSelector(prestoActionConfig.getJdbcUrls()),
+                prestoActionConfig,
```

Comment:
Should we add a test to make sure that we actually cycle over the jdbc urls?

Reply:
I'll merge this to unblock the fix first.